docs/web-wallet.md

```diff
@@ -0,0 +1,254 @@
+2. **Connecting the Wallet**: Upon arrival, the user selects 'Connect Wallet' and connects their Ethereum wallet with the tool.
+3. **Creating Identity and Accessing Credentials**: Next, the user selects 'Sign Message' to create a derived identity wallet and retrieve the encrypted credentials associated with their Ethereum wallet via the cloud storage.
+4. **Claiming the Credential**: If the user hasn't previously claimed the credential required by the verifier’s request, they proceed by clicking 'Claim Credential' and complete the issuance process (a face scan in this scenario) to claim the credential via the embedded issuance.
+5. **Proof generation**: Once the user owns the credential, can select 'Verify', triggering the generation of the zk-proof which is then shared to the verifier application in case of off-chain verification and to the smart contract in case of on-chain verification
+6. **Verification Completion**: The user finally clicks the ‘Continue’ button, upon which they are redirected to the application where the verification can be completed and the application flow can continue.
+
+### End-User Journey of Verification of other credentials
+
+When proving custom authentication on an application:
+1. **Initiating the Process**: The user clicks a designated button, initiating redirection to the Web Wallet.
+2. **Connecting the Wallet**: Upon arrival, the user selects 'Connect Wallet' to connect their ethereum wallet with the tool.
+3. **Creating Identity and Accessing Credentials**:  Next, the user selects 'Sign Message' to create a derived identity wallet and retrieve the encrypted credentials associated with their Ethereum wallet via the cloud storage.
+4. **Requesting mobile verification**: The user clicks the “Verify from mobile” button and a QR code is presented to start the verification from a mobile device.
+5. **Generating Proof via the Polygon ID App**: The user then utilizes the Polygon ID App (where they already have claimed the credential) to scan the displayed QR code, initiating proof generation and commencing the verification process.
+6. **Verification Completion**: The user finally clicks the ‘Continue’ button , upon which they are redirected to the application where the verification can be completed and the application flow can continue. 
+
+## Integration Process for the Verifiers
+
+### Step 1 - Choosing Your Credential Schema
+
+Select the credential schema that aligns with your requirements from our [Schema Explorer](https://schema-builder.polygonid.me/) or any other source.
+Standardized schemas, such as KYC, are available. Alternatively, you can create a custom schema using our [Schema Builder](https://schema-builder.polygonid.me/builder).
+
+### Step 2 - Creating the Query
+
+Define what information you need from your users and build a query using the Query Builder [https://schema-builder.polygonid.me/query-builder ].
```

Comment:
This link fails to render due to the last space.

Reply:
fixed